sdk/lusid/models/weighted_instrument.py

```diff
@@ -44,24 +45,11 @@ class WeightedInstrument(object):
         'instrument': 'instrument'
     }
 
-    required_map = {
-        'quantity': 'optional',
-        'holding_identifier': 'optional',
-        'instrument': 'optional'
-    }
-
-    def __init__(self, quantity=None, holding_identifier=None, instrument=None):  # noqa: E501
-        """
-        WeightedInstrument - a model defined in OpenAPI
-
-        :param quantity:  The quantity of the instrument that is owned.
-        :type quantity: float
-        :param holding_identifier:  Identifier for the instrument.  For a single, unique trade or transaction this can be thought of as equivalent to the transaction identifier, or  a composite of the sub-holding keys for a regular sub-holding. When there are multiple transactions sharing the same underlying instrument  such as purchase of shares on multiple dates where tax implications are different this would not be the case.    In an inlined aggregation request if this is wanted to identify a line item, it can be specified in the set of aggregation keys given on the aggregation  request that accompanies the set of weighted instruments.
-        :type holding_identifier: str
-        :param instrument: 
-        :type instrument: lusid.LusidInstrument
```

Comment:
It looks like this init docstring with the types and description of each parameter have also been moved into the getter and setter for each attribute, what does this mean for a user of an IDE such as PyCharm? Will they still be able to easily see what types to provide when creating a model?

Reply:
Should be back to how it was before now hopefully

Reply:
Is that better?

Reply:
Can we add to the new mustache template this information?